lib/hanami/cli.rb

```diff
@@ -1,37 +1,66 @@
-require "hanami/cli/version"
-require "hanami/cli/renderer"
-require "hanami/cli/command/params_parser"
-
 module Hanami
   class Cli
-    attr_reader :commands
-    attr_reader :renderer
+    require "hanami/cli/version"
+    require "hanami/cli/command"
+    require "hanami/cli/registry"
+    require "hanami/cli/parser"
+    require "hanami/cli/usage"
+    require "hanami/cli/banner"
```

Comment:
I'm curious to learn why your policy for `require` lines like these is to nest them within the relevant module. I've always done it at the top of the file, but that's really because "it's always how I've done it". Would love to learn your perspective on this :)